mock-identity-system/src/main/java/io/mosip/esignet/mock/identitysystem/service/IdentityService.java

```diff
@@ -11,4 +17,6 @@ public interface IdentityService {
 	public IdentityData getIdentity(String individualId) throws MockIdentityException;
 
     public  void addVerifiedClaim(VerifiedClaimRequestDto request) throws MockIdentityException;
+
+	public JsonNode getIdentityInJsonNode(String individualId) throws MockIdentityException;
```

Comment:
Kindly rename the method

Reply:
changed it to getIdentityV2 as method overloading is not possible with different return type